### PR TITLE
feat : Added different types of cards.

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
- Card with badge on it.
- Card with dismiss option.
- Card with text overlay.
- Card with text only.
- Vertical and Horizontal card.
- Cards with shadow.

### CODE REVIEW

**Comment by shuklahimanshu81** [comment]
Screenshot (335)

**Comment by shuklahimanshu81** [comment]
Screenshot (336)

File: components/Cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+.about-card{
+  display: flex;
+  flex-direction: column;
+}
+.dismiss-btn{
+    background-color: var(--white);
+    border-radius: var(--border-radius-f);
+    border: none;
+    bottom: 24rem;
+    color:gray;
```
Comment (stutisk) [inline components/Cards/cards.css:90]: can add this to a variable

**Review by stutisk — COMMENTED** [review]
this looks good to merge

File: components/Cards/cards.css (hunk 1 above)
Comment (shuklahimanshu81) [reply components/Cards/cards.css:90]: Will do this while refactoring.

File: components/Cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    font-size: calc(var(--text-size-s) + 4px);
+}
+.card-detail{
+   font-size: var(--text-size-m);
+}
+.card-btn{
+    border-radius: var(--border-radius-l);
+    font-size: var(--text-size-s);
+    height: 2rem;
+    padding: 0.25rem 0.5rem;
```
Comment (Radheshyam1234) [inline components/Cards/cards.css:41]: use px unit instead of rem for value <=10px

File: components/Cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    box-shadow: none;
+    opacity: 0.5;
+}
+.text-overlay{
+    position: relative;
+    bottom: 14.5rem;
+    width:17.5rem ;
+    background-color: var(--gray);
+    font-size: var(--text-size-m);
+    font-weight: 600;
```
Comment (Radheshyam1234) [inline components/Cards/cards.css:115]: I would like to suggest that for font weight you could also define a variable like font-bold, font-semi-bold, font-regular  and  could use this within your project.

File: components/Cards/cards.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+.card-secondary-btn{
+    border: 2.5px solid var(--primary-color);
+    color: var(--primary-color);
+}
+.vertical-card{
+    box-sizing: border-box;
+}
+.card-badge{
+    background-color: var(--secondary-color);
+    border-top-right-radius: var(--border-radius-f);
```
Comment (MaruthiPothuganti) [inline components/Cards/cards.css:58]: You can use short hand property for this

File: components/Cards/cards.css (hunk 3 above)
Comment (shuklahimanshu81) [reply components/Cards/cards.css:115]: Will do while refactoring next time.

**Review by MaruthiPothuganti — APPROVED** [review]

**Pull request merged**